staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/validation/cel/compilation.go

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
split code changes from vendoring

Reply:
Oops, I've rebased and completely reorganized the commits for readability.